auth/auth.go

```diff
@@ -142,11 +142,6 @@ func (auth *Auth) Configure(config core.ServerConfig) error {
 		StrictMode:            config.Strictmode,
 	}, auth.vcr, resolver.DIDKeyResolver{Resolver: auth.vdrInstance.Resolver()}, auth.keyStore, auth.jsonldManager, auth.pkiProvider)
 
-	tlsEnabled := config.TLS.Enabled()
-	if config.Strictmode && !tlsEnabled {
-		return errors.New("in strictmode TLS must be enabled")
-	}
```

Comment:
does this check still exists in the n2n flow somewhere?

Reply:
It's performed by the strict HTTP client, which checks the request URL scheme (HTTP/HTTPS)